tests/auth/test_authorization.py

```
@@ -296,57 +252,66 @@ async def test_authenticate_jwt_user_not_found(self) -> None:
-        settings for the test case.
-
-        Sets self.client to a TestClient created from the application
-        and sets self.auth_settings with a test JWT secret, algorithm,
-        and access token expiry (3600 seconds).
-        """
-        self.client = testclient.TestClient(app.create_app())
+        """Prepare TestClient and auth settings."""
+        self.app = app.create_app()
+        self.client = fastapi.testclient.TestClient(self.app)
         self.auth_settings = settings.Auth(
             jwt_secret='test-secret-key-32-characters!',
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to argument: "jwt_secret"

(S106)